.github/workflows/tests.yml

```
@@ -141,6 +141,35 @@ jobs:
+          sudo ln -sf /tmp/resource /usr/share/infinity/resource
+          mkdir -p resource
+          ln -sf /tmp/resource/wordnet resource/wordnet
+
+      - name: Unit tests
+        run: |
+          set -euo pipefail
+          CGO_ENABLED=1 GOPROXY=${GOPROXY:-https://goproxy.cn,https://proxy.golang.org,direct} \
+            go test -count=1 ./internal/...
+
+      - name: Integration tests
+        if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)